### DISCUSSION TITLE
Select row (in ui.table) without "selection checkbox" - only with click

### DISCUSSION THREAD

**Discussion by bobanovo** [zauberzeug/nicegui#809] [Q&A] [answered]
### Question

Hi, all,
pls, is it possible to select row (and use selection in function then) in ui.table only with click to specific row?
thanks a lot for any help :-), 
br bo

**Comment by rodja** [comment, marked as answer]
Good question. You are not alone. See https://github.com/zauberzeug/nicegui/discussions/802#discussioncomment-5681712 for some pointers. It currently hinges on the implementation of #672.

**Reply by falkoschindler** [reply]
See also #774.

**Reply by bobanovo** [reply]
Hi, thanks. I would like to use ui.table. For this, I found the solution, which works for me - as emitting my own event on the q-tr in ui-table as:

```py
table1.add_slot('body', """
	<q-tr :props="props" @click="">
	<q-td key="name" :props="props"	@click="$parent.$emit('action', props.row.id)"
		:props="props"
		:class="(props.row.status!='ACTIVE')?'bg-accent text-white':'bg-white text-black'"
	>

...
table1.on('action', handle_action)	
```

**Reply by falkoschindler** [reply]
@bobanovo Wow, that's a great idea! I posted a complete example using your approach here: https://github.com/zauberzeug/nicegui/discussions/774#discussioncomment-5700296
